### PR TITLE
Add logging

### PR DESCRIPTION
**Pull request by kalbermattenm**

### CODE REVIEW

**Review by maltaesousa — APPROVED** [review]
Nice! Maybe you want to add a line to the Dockerfile: `ENV PYTHONUNBUFFERED=1`, this will make the logging visible in docker-compose logs ;)
Example here:
https://github.com/maltaesousa/CapitalGuessr/blob/main/back/Dockerfile

File: README.md [hunk 1]
```diff
@@ -43,6 +43,38 @@ docker-compose down --remove-orphans -v
 docker-compose up --build    
 ```
 
+## Using Windows WSL
```
Comment (monodo) [inline README.md:46]: :broken_heart:

**Comment by monodo** [comment]
nice :-)

**Review by monodo — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (kalbermattenm) [reply README.md:46]: Don't cry, it is only for development purposes...

**Comment by kalbermattenm** [comment]
I am merging this.

**Pull request merged**